settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "client_id": "client-1",
   "client_secret": "Password1",
-  "audience": "client-one",
+  "audience": "client-1",
```

Comment:
why this?

Reply:
The config for that client on nordiacpis.curity.io is with `client-1` as audience.

Reply:
but it won't work for default setups -- like demos, sales calls, etc. `audience` can be an array, maybe we should set the default to be `["client-1", "client-one]`.